src/aosm/azext_aosm/cli_handlers/onboarding_base_handler.py

```diff
@@ -175,11 +179,16 @@ def _read_input_config_from_file(input_json_path: Path) -> dict:
 
         Returns config as dictionary.
         """
-        lines = input_json_path.read_text().splitlines()
-        lines = [line for line in lines if not line.strip().startswith("//")]
-        config_dict = json.loads("".join(lines))
-
-        return config_dict
+        try:
+            lines = input_json_path.read_text().splitlines()
+            lines = [line for line in lines if not line.strip().startswith("//")]
+            config_dict = json.loads("".join(lines))
+            return config_dict
+        except (UserFault, FileNotFoundError) as e:

```

Comment:
I'm surprised `UserFault` needs to be caught here. What's raising it?

Reply:
leftover from when i was thinking not providing jsonc should fail, but all good now so will remove